src/helpers/format.ts

```diff
@@ -168,6 +171,10 @@ function applyInternalFormat(
 }
 
 function applyInternalNumberFormat(value: number, format: InternalNumberFormat, locale: Locale) {
+  if (value === Infinity) {
```

Comment:
also needs to handle `-Infinity`

Reply:
No, this function is called with Math.abs(value), and after a "minus" sign is add to the result in case of negative value